test_unstructured_ingest/test-ingest.sh

```diff
@@ -91,3 +91,7 @@ for test in "${all_tests[@]}"; do
     echo "--------- FINISHED SCRIPT $test ---------"
   fi
 done
+
+echo "--------- RUNNING SCRIPT evaluation-metrics.sh ---------"
+./test_unstructured_ingest/evaluation-metrics.sh
+echo "--------- FINISHED SCRIPT evaluation-metrics.sh ---------"
```

Comment:
please add newline at end of file per github "complaint" above (same for other files where applicable)